src/test/java/co/org/osso/api/ApiApplicationTests.java

```diff
@@ -1,16 +1,40 @@
 package co.org.osso.api;
 
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.util.List;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class ApiApplicationTests {
 
+	@Autowired
+	private BookController controller_target;
```

Comment:
Let's split the tests for the `Book` stuff into it's own test file, and the `Customer` tests in another file.

Reply:
Done. I hope... :)